### PR TITLE
feat: sticky headers

### PR DESCRIPTION
**Pull request by yildirayunlu**
feat: sticky headers

### CODE REVIEW

**Review by alicanerdurmaz — APPROVED** [review]

File: .changeset/neat-snakes-sit.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+"@refinedev/chakra-ui": minor
+---
+
+feat: add `isSticky` prop to `ThemedHeaderV2` component
+
+```tsx
+import { ThemedLayoutV2, ThemedHeaderV2 } from '@refinedev/chakra-ui';
```
Comment (salihozdemir) [inline .changeset/neat-snakes-sit.md:8]:
```suggestion
import { ThemedLayoutV2 } from '@refinedev/chakra-ui';
```

File: .changeset/angry-pots-pretend.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+"@refinedev/antd": minor
+---
+
+feat: add `isSticky` prop to `ThemedHeaderV2` component
+
+```tsx
+import { ThemedLayoutV2, ThemedHeaderV2 } from '@refinedev/antd';
```
Comment (salihozdemir) [inline .changeset/angry-pots-pretend.md:8]:
```suggestion
import { ThemedLayoutV2 } from '@refinedev/antd';
```

File: .changeset/nine-boats-look.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+"@refinedev/mantine": minor
+---
+
+feat: add `isSticky` prop to `ThemedHeaderV2` component
+
+```tsx
+import { ThemedLayoutV2, ThemedHeaderV2 } from '@refinedev/mantine';
```
Comment (salihozdemir) [inline .changeset/nine-boats-look.md:8]:
```suggestion
import { ThemedLayoutV2 } from '@refinedev/mantine';
```

**Review by salihozdemir — APPROVED** [review]

**Review by aliemir — CHANGES_REQUESTED** [review]
The changeset for `@refinedev/ui-types` is missing @yildirayunlu

**Review by aliemir — CHANGES_REQUESTED** [review]
Since we've added the `isSticky` prop to the `RefineThemedLayoutV2HeaderProps` type. We should also update the `@refinedev/mui`'s `ThemedSiderV2` to reflect the changes accordingly. (By default the `AppBar` was set to `position: sticky`)

@yildirayunlu Thank you 🙏

**Review by alicanerdurmaz — APPROVED** [review]

**Review by aliemir — APPROVED** [review]

**Pull request merged**